marketplace_confg/chart/dicom-adapter/templates/application.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "{{ .Release.Name }}"
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    app.kubernetes.io/name: "{{ .Release.Name }}"
+  annotations:
+    marketplace.cloud.google.com/deploy-info: '{"partner_id": "", "product_id": "", "partner_name": ""}'
+spec:
+  selector:
+    matchLabels:
+     app.kubernetes.io/name: "{{ .Release.Name }}"
+  componentKinds:
+    - group: v1
+      kind: Service
+    - group: apps/v1
+      kind: StatefulSet
+  addOwnerRef: true
```

Comment:
What does this do?

Reply:
componentKinds is used to indicate the type of resources that the instance is using. This link provides details of the application object (required by Marketplace): https://github.com/kubernetes-sigs/application/blob/master/docs/api.md

Reply:
Thanks for the link